### PR TITLE
Style Rwanda profile view and edit pages

### PR DESCRIPTION
**Pull request by KaitCrawford**
This updates the style of the profile view and edit pages as well as the change password page.

### CODE REVIEW

**Comment by KaitCrawford** [comment]
View Profile page:
profile_view

Edit profile page:
edit_profile_1
profile_edit_2

Change pin page:
change_pin_page

File: gem/templates/rwanda/profiles/viewprofile.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  <div class="profiles">
+    <div class="profiles__account"
+    <fieldset>
+      <label class="profiles__description--label">{% trans "When you type comments your name will appear as:" %}</label>
+      <p class="profiles__description profiles__alias">{{ user.profile.alias | default:" Anonymous " }}</p>
+    </fieldset>
+
+    {% trans "Not set." as defaultText %}
+    <fieldset>
+      <label class="profiles__description--label">{% trans "Date of Birth: " %}</label>
```
Comment (Mitso) [inline gem/templates/rwanda/profiles/viewprofile.html]: @KaitCrawford is there a specific reason the classes to modify profiles__description has different name? They could just use the same modifier class

File: gem/styles/gem-rwanda/modules/_m-profiles.scss [hunk 2]
```diff
@@ -27,23 +27,31 @@
     }
   }
   &__account {
-    padding: 0 $spacer-mid;
+    padding: $spacer-small $spacer-mid;
+    & .call-to-action{
```
Comment (Mitso) [inline gem/styles/gem-rwanda/modules/_m-profiles.scss]: can remove the & here as well and just use .call-to-action

File: gem/templates/rwanda/profiles/viewprofile.html (hunk 1 above)
Comment (KaitCrawford) [reply gem/templates/rwanda/profiles/viewprofile.html]: @Mitso I'm afraid I don't understand. Which modifier class should they be using? (tbh I'm not sure what a 'modifier class' is)

**Comment by Mitso** [comment]
@KaitCrawford looks good to me. 🍰 🍰 🍰

**Pull request merged**